firmware_configurations/README.md

```diff
@@ -23,6 +23,7 @@ If your mod is lacking a column in the compatibility matrix, feel free to submit
 
 | Creator | Config title | Description | V1.6 | V2.1 | V2.2 |
 |---------|--------------|-------------|------|------|------|
+| bryansj  | [SKR 1.3 V2.2 Extra Settings for Reference](./klipper/eddie) | Voron Design VORON2.2 300mm SKR 1.3 TMC2209 config with Extra Features Enabled for Reference | :x: | :grey_question: | :heavy_check_mark: |
```

Comment:
This links to the wrong folder (./klipper/eddie)

Reply:
copy and paste error...